third_party/2and3/click/testing.pyi

```diff
@@ -20,25 +20,35 @@ class Result:
     exit_code: int
     exception: Any
     exc_info: Optional[Any]
+    stdout_bytes: bytes
+    stderr_bytes: bytes
+    # properties
```

Comment:
Should be given with `@property`. It matters because this way type checkers can see that assigning to `.output` is an error.

Reply:
Thanks for the tip, I didn't consider that.

I noticed this pattern in a few other places, like in [parser.py](https://github.com/python/typeshed/blob/master/third_party/2and3/click/parser.pyi#L33-L34) and [core.py](https://github.com/python/typeshed/blob/master/third_party/2and3/click/core.pyi#L68-L70). I wonder if they also need fixing or if I am missing some subtle detail?

Reply:
I haven't checked these against the implementation, but if those are also `@property`s without setters, they should use `@property` in the stub. (If they do have a setter, it's fine to just represent them in the stub as you did here.)

Reply:
Thanks! I guessed so, but wanted to double check.

I have submitted https://github.com/python/typeshed/pull/3028 to fix that.